README.md

```diff
@@ -1,2 +1,47 @@
-# dell-terraform-providers
-Terraform providers will provide easy and seamless integration with Dell Infrastructure.
+<!--
+Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Dell Terraform Providers
+
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](docs/CODE_OF_CONDUCT.md)
+[![License](https://img.shields.io/badge/License-MPL_2.0-blue.svg)](LICENSE)
+
+Dell Terraform providers is an open-source suite of Terraform providers for Dell products.
+
+For documentation, please visit [Dell Terraform Providers documentation](https://dell.github.io/terraform-docs/).
+
+## Table of Contents
+
+* [Code of Conduct](./docs/CODE_OF_CONDUCT.md)
+* [Maintainer Guide](./docs/MAINTAINER_GUIDE.md)
+* [Committer Guide](./docs/COMMITTER_GUIDE.md)
+* [Contributing Guide](./docs/CONTRIBUTING.md)
+* [Branching Strategy](./docs/BRANCHING.md)
+* [List of Adopters](./docs/ADOPTERS.md)
```

Comment:
Check ADOPTERS.md file

Reply:
I think this is okay. We do not have any adopters yet in any case